api/include/opentelemetry/trace/trace_state.h

```diff
@@ -1,96 +0,0 @@
-// Copyright 2020, OpenTelemetry Authors
```

Comment:
Is there a reason this file needs to be deleted? I don't think `TraceState` is referenced anywhere in the rest of this PR (except in comments).